token/token_test.go

```diff
@@ -1 +1,314 @@
-package token_test
+package token
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/EnAccess/OpenPAYGO-go/token/shared"
+	"testing"
+)
+
+type TTokenContext struct {
+	Context TokenContext
+	Token   string
+}
+
+type TokenJSON struct {
+	SerialNumber       string `json:"serial_number"`
+	StartingCode       uint32 `json:"starting_code"`
+	Key                string `json:"key"`
+	TokenCount         uint8  `json:"token_count"`
+	RestrictedDigitSet bool   `json:"restricted_digit_set"`
+	TokenType          int    `json:"token_type"`
+	ValueRaw           int    `json:"value_raw"`
+	Token              string `json:"token"`
+}
+
+func getTestTokens() ([]TTokenContext, error) {
```

Comment:
Would it be possible to read the file from disk? In the future I would like to update the test cases with even more and make sure all OpenPAYGO libraries use exactly the same file.

Reply:
Yes, should be able to add this in I believe.

Reply:
Updated.